tools-core/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("com.hedera.hashgraph.java-conventions")
+    id("org.openjfx.javafxplugin") version "0.0.13"
+}
+
+dependencies {
+    implementation(libs.bundles.logging)
+    implementation(libs.bundles.bouncycastle)
+    implementation(libs.jackson.databind)
+    implementation(libs.eddsa)
+    implementation(libs.protobuf.javalite)
+    implementation(libs.protobuf.java.util)
+    implementation(libs.commons.codec)
+    implementation(libs.commons.io)
+    implementation(libs.httpclient)
+    implementation(libs.zt.zip)
+    implementation(libs.opencsv)
+    implementation(libs.picocli)
+//    implementation(libs.json)
+    implementation(libs.grpc.netty)
+    implementation(libs.maven.artifact)
+    implementation(libs.passpol)
+    implementation(libs.jetbrains.annotations)
+    testImplementation(testLibs.junitparams)
+    testImplementation(testLibs.testfx.junit)
+    testImplementation(testLibs.bundles.junit)
+//    testRuntimeOnly(testLibs.junit.jupiter.engine)
+    testRuntimeOnly(testLibs.testfx.openjfx.monocle)
+}
+
+javafx {
+    version = "19"
```

Comment:
we should define the JFX version globally

Reply:
+ why is core depending on JFX? should only be used in tools-ui

Reply:
Not sure, to be honest. I haven't begun cleaning core yet, as tools-ui was an easier place for me to jump in and start cleaning things up. This was left over from the maven build. I'll see about removing it.

Reply:
It appears that the RemoteFile subclasses are used to create their own little gridPanes, and so need JFX. I will be fixing that in the future, but the dependency/plugin is still needed until then.